datatypes/string.md

```diff
@@ -0,0 +1,44 @@
+                                                    Python strings.

```

Comment:
Write:
# Python Strings
Instead of 
` Python Strings.

Keep it professional